downloader/history_downloader/history_manager.py

```diff
@@ -0,0 +1,40 @@
+import ccxt
+import time
+
+
+class HistoryDownloader:
+
+    def __init__(self, exchange_name: str, crypto_symbol: str, days_back: int):
+        """ init
+            Paramaters
+            ------------
+            exchange_name: str required - name of cryptocurrency, e.g. 'binance'
+            crypto_symbol: str required - symbol of cryptocurrency, e.g. in Bitcoin in Binance is 'BTC/USDT'
+            days_back: str required  - the number of days back from which we want to retrieve the data
+        """
+        self.exchange_name = exchange_name.lower()
+        self.crypto_symbol = crypto_symbol
+        self.exchange = getattr(ccxt, self.exchange_name)()  # creating instance of an exchange
+        self.days_back = days_back * 2
+        # number of days are multiplied by 2 because we divide each day into two twelve-hour periods
+
+    def get_history(self) -> list:
+        """ downloading cryptocurrency history
+                max. candle limit is 1000, therefore this function
+                splits 1 day into 2 twelve-hour periods
+                each period provides 720 candles
+         """
+        current_miliseconds = int(round(time.time() * 1000))
+        if self.exchange.has['fetchOHLCV']:
+            history_data = []
+            for i in range(1, self.days_back + 1):
+                candles = self.exchange.fetch_ohlcv(self.crypto_symbol, '1m', limit=720,
+                                                    params={'startTime': current_miliseconds - (
+                                                            43200000 * (self.days_back - i + 1)),

```

Comment:
Putting equesions into a function callback is not readable, it would be better to have this equastion described separetly eg:
```python
# below eq is doing...
var_1 = 98875 * self.param / 6 * (4 + self.param) ... 
func.call(var_1)
```

Reply:
Yes, it's actually cleaner. Fixed it